### PR TITLE
chore: some changes to script 1

### PR DESCRIPTION
**Pull request by derberg**
comments inline to explain my feedback

### CODE REVIEW

File: VIDEO_1_SCRIPT.md
Comment (derberg) [inline VIDEO_1_SCRIPT.md:68]: purely editorial, I use proper makrdown syntax to better render bullet points, with proper indentation

File: VIDEO_1_SCRIPT.md
Comment (derberg) [inline VIDEO_1_SCRIPT.md:74]: kinda, don't know but somehow I think we need this clarification, maybe also some text should be in previous section that ok, there are 3 types of APIs we will not expand in training on Library and Function APIs

File: VIDEO_1_SCRIPT.md
Comment (Barbanio) [reply VIDEO_1_SCRIPT.md:68]: Great, thanks! :)

File: VIDEO_1_SCRIPT.md
Comment (Barbanio) [inline VIDEO_1_SCRIPT.md:73]:
```suggestion

We have just mentioned the Library and Function APIs, but we are going to dive into the Web APIs since they are the most relevant to AsyncAPI. 

```

How about saying this?

File: VIDEO_1_SCRIPT.md
Comment (Barbanio) [reply VIDEO_1_SCRIPT.md:74]: Yes, it makes sense. I proposed a suggestion about this above.

File: VIDEO_1_SCRIPT.md
Comment (derberg) [reply VIDEO_1_SCRIPT.md:73]: sure, but I think we can skip `since they are the most relevant to AsyncAPI.`
- we do not have to explain ourselves 😄 
- first training videos are a general purpose, people do not need to understand what "AsyncAPI" means, so maybe we should not introduce that term

Comment (Barbanio) [reply VIDEO_1_SCRIPT.md:73]: Ok, ready!

**Comment by derberg** [comment]
are we waiting for review from @alequetzalli ?

**Comment by Barbanio** [comment]
Yes 🙂

**Comment by Barbanio** [comment]
I think so 🙂

**Review by alequetzalli — APPROVED** [review]

**Pull request merged**
